base/logging.hpp

```diff
@@ -25,6 +25,19 @@ namespace my
 
   void LogMessageDefault(LogLevel level, SrcPoint const & srcPoint, string const & msg);
   void LogMessageTests(LogLevel level, SrcPoint const & srcPoint, string const & msg);
+
+  /// Scope Guard to temporarily suppress specific log level, for example, in unit tests:
+  /// ...
+  /// {
+  ///   LogLevelSuppressor onlyLERRORAndLCriticalLogsAreEnabled;
+  ///   TEST(SomeFunctionWhichHasDebugOrInfoOrWarningLogs(), ());
+  /// }
+  struct LogLevelSuppressor
```

Comment:
Both names have pros and cons. ScopedLogLevel does not tell much about it's use case: temporarily change good default behavior.